packages/admin-stories/src/admin/helpers/DataGenerators.ts

```diff
@@ -0,0 +1,34 @@
+export const generateRandomEmails = (amount: number, valueLength: number) => {
```

Comment:
the file should not start with an upper case letter

Reply:
will change that :)